topics/calculate-pi/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: calculatepi, calculate-pi
+created_by: github
+display_name: Calculate Pi
+github_url: https://github.com/topics/calculate-pi
+# logo: raspberry-pi.png
+released: March 14, 1592
+short_description: The mathematical Constant π
+topic: Calculate Pi
+url: https://en.wikipedia.org/wiki/Pi
+wikipedia_url: https://en.wikipedia.org/wiki/Pi
+---
+
+The number π (/paɪ/) is a mathematical constant. Originally defined as the ratio of a circle's circumference to its diameter, it now has various equivalent definitions and appears in many formulas in all areas of mathematics and physics. It is approximately equal to 3.14159. It has been represented by the Greek letter "π" since the mid-18th century, though it is also sometimes spelled out as "pi". It is also called Archimedes' constant.
```

Comment:
```suggestion
The number π (/paɪ/) is a mathematical constant. Originally defined as the ratio of a circle's circumference to its diameter, it now has various equivalent definitions and appears in many formulas in all areas of mathematics and physics. It is approximately equal to 3.14159. It has been represented by the Greek letter "π" since the mid-18th century, though it is also sometimes spelled out as "pi". It is also called Archimedes' constant. [Wikipedia](https://en.wikipedia.org/wiki/Pi).

If you are up to the challenge, this topic contains repositories with code to do just that.
```

Added the last line and a source link to Wikipedia (just in case).